### ISSUE TITLE
Validation split

### ISSUE THREAD

**Issue by jonathancstroud** [MichiganDataScienceTeam/googleanalytics#2]
Need a reproducible way to split `train.csv` into a training and validation set.

**Comment by ShuaoZhang** [comment]
It's quite easy if we only have to split the data. It can be done in one line with ***sklearn.model_selection.train_test_split***., for example.

However, when we do split, we should consider the feature in the data set.  For example, "Chrome" has  a large ratio in the "device" column, so we'd better split with that ratio. Unfortunately, it's really hard to determine before we have a good knowledge about the data. 

So shall we just split in a simple way at first? Or we look at the data at first?

**Comment by jonathancstroud** [comment]
As you suggest, we need to look at the data. We want a validation set that is representative of the test set, so it would be good to have some checks that show that the split gives us a good balance of features, in case this doesn't happen naturally from a random split.

Also important, we need to consider that repeat visitors may (or may not) be represented in both the training and test set. I'm not sure if the split is done over fullVisitorId or visitId. For our validation set, we should match whatever procedure they use to split train and test.

It's worth checking - are all the visitors in the training set unique from those in the test set?

**Comment by jonathancstroud** [comment]
Closed by #97
